cli/js/unit_test_runner.ts

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env -S deno run --reload --allow-run
+#!/bin/sh
+":"; //; exec /usr/bin/env deno run --reload --allow-run "$0" "$@"
```

Comment:
I know this is for the Prettier situation, but essentially this first `;` renders `":"` useless. Maybe just remove it altogether.

Also maybe `// 2> /dev/null;` just in case for garbage generated

Reply:
True, it is sad. I've made it a call expression to make the formatter (and shells) happy.